verilog/tools/preprocessor/verilog_preprocessor_test.sh

```diff
@@ -560,5 +569,241 @@ diff --strip-trailing-cr -u "$MY_EXPECT_FILE" "$MY_OUTPUT_FILE" || {
   exit 1
 }
 
+################################################################################
+echo "=== Test multiple-compilation-unit: including a file with an absolute path"
+
+cat > "$MY_INPUT_FILE" <<EOF
+\`include "${MY_ABSOLUTE_INCLUDED_FILE_1}"
+input_content
+EOF
+
+cat > "$MY_ABSOLUTE_INCLUDED_FILE_1" <<EOF
+included1_content
+EOF
+
+
+cat > "$MY_EXPECT_FILE" <<EOF
+${MY_INPUT_FILE}:
+(#293: "included1_content")
+(#293: "input_content")
+
+EOF
+
+"$preprocessor" multiple-compilation-unit "$MY_INPUT_FILE" > "$MY_OUTPUT_FILE" 2>&1
+
+status="$?"
+
+[[ $status == 0 ]] || {
+  "Expected exit code 0, but got $status"
+  exit 0
```

Comment:
Wouldn't we want tto `exit 1` here as we want to indicate the failure to match the expectation ?
(also other places below)

Reply:
Yes, I didn't want to change this, the idea here is that the `"Expected exit..."` will give a `no command found` error, because it's missing `echo`.